addons/sourcemod/scripting/shavit-mapchooser.sp

```diff
@@ -45,6 +45,9 @@ int gI_Driver = Driver_unknown;
 Database g_hDatabase;
 char g_cSQLPrefix[32];
 
+//Jacob added this
+public bool instantChange = false;
```

Comment:
A global bool is probably not needed and anyway it should probably follow similar naming conventions as other global variables (prefixed with `g_b`)